src/latexgen.cpp

```diff
@@ -497,7 +497,7 @@ static void writeDefaultHeaderPart1(FTextStream &t)
     documentClass = "book";
   t << "\\documentclass[twoside]{" << documentClass << "}\n"
        "\n";
-  t << "%% moved from doxygen.sty due to workaround for LatEx 2019 version and unmaintained tabu package\n"
+  t << "%% moved from doxygen.sty due to workaround for LaTex 2019 version and unmaintained tabu package\n"
        "\\usepackage{ifthen}\n"
        "\\ifx\\requestedLaTeXdate\\undefined\n"
```

Comment:
I'm not 100% confident about this construct. I think it would be safer to also include the matching version of array (as array_doxygen) along with doxygen. Then this can also be placed back in doxygen.sty

Reply:
I don't think so, this was suggested by the LaTeX group members as it is provided as part of the array  package (and I also wanted it to load as one of the first so that no other package could load another version of the array package).